libraries/rtprocessing/rtfilter.h

```diff
@@ -109,15 +110,47 @@ class RTPROCESINGSHARED_EXPORT RtFilter
     /**
     * Calculates the filtered version of the raw input data
     *
-    * @param [in] matDataIn     data which is to be filtered
-    * @param [out] matDataOut    data which is to be filtered
-    * @param [in] iDataIndex    current position in the global data matrix
+    * @param [in] matDataIn - data which is to be filtered
```

Comment:
Please remove all - in variable description. For example " matDataIn - data which is to be filtered" should be "matDataIn The data which is to be filtered". Check all other new code as well.